.changeset/slow-houses-develop.md

```diff
@@ -2,4 +2,4 @@
 'sku': patch
 ---
 
-Improve logging for `sku translations` CLI commands
+`sku translations`: Improve log messages
```

Comment:
**question (non-blocking):** Is it possible be more descriptive on what exactly has improved about the log messages?

Is this TMI or would it be helpful to state a bit more detail on what has improved?

Reply:
For this one it felt like TMI. https://github.com/seek-oss/sku/pull/1075
It was a timing thing due to not awaiting some async functions. I can try improve the messaging a bit.

Reply:
https://github.com/seek-oss/sku/pull/1091/commits/6cb9b2c3205987c06f284687e52517f08ad47189

Reply:
**Suggestion (non-blocking):** In the linked PR I saw that line on 47 in the `packages/sku/scripts/translations.js` file you removed `sku.config.js` from the log. The new log text reads weird without that context missing.

Probably something for another PR, just wanted to point it out.